deepmd/pd/entrypoints/main.py

```
@@ -0,0 +1,606 @@
+            output_path,
+            {},
+        )
+    log.info(f"Saved model to {output_path}")
+
+
+# @record
+def main(args: Optional[Union[List[str], argparse.Namespace]] = None):
+    if not isinstance(args, argparse.Namespace):
+        FLAGS = parse_args(args=args)
+    else:
+        FLAGS = args
```

Ruff

Use ternary operator `FLAGS = parse_args(args=args) if not isinstance(args, argparse.Namespace) else args` instead of `if`-`else`-block

Replace `if`-`else`-block with `FLAGS = parse_args(args=args) if not isinstance(args, argparse.Namespace) else args`

(SIM108)